setup.py

```diff
@@ -1,22 +1,27 @@
+"""Setup module for the myco-api-client."""
+
 import os
-from setuptools import setup, find_packages
+
+from setuptools import find_packages, setup
+
 from myco_api_client import __version__
 
-target_branch = os.environ.get("BRANCH", "master")
+TARGET_BRANCH = os.environ.get("TARGET_BRANCH", "master")
 
 
 try:
-    with open('requirements/base.txt') as req:
-        REQUIREMENTS = [r.partition('#')[0] for r in req if not r.startswith('-e')]
+    with open("requirements/base.txt", encoding="utf-8") as req:
```

Comment:
Why did you need to set the encoding here? Did someone try to run `fab sync` in windows and did not work out ?

Reply:
Pylint warns that assuming that the default locale is always utf-8 might result in bugs (especially on windows, as you mentioned):
> pylint: warning
W1514 - Using open without explicitly specifying an encoding (unspecified-encoding)

I changed it to avoid bugs, but also pylint warnings :)
Here's the related [PEP 597](https://www.python.org/dev/peps/pep-0597/#motivation)